DOCS.md

```diff
@@ -63,6 +63,38 @@ drone secret add \
 
 Kubernetes expects secrets to be base64 encoded, `drone-gke` does that for you. If you pass in a secret that is already base64 encoded, please apply the prefix `secret_base64_` and the plugin will not re-encode them.
 
+## Available vars
+
+These variables are always available to reference in any manifest, and cannot be overwritten by `vars` or `secrets`:
+
+```
+---START VARIABLES AVAILABLE FOR ALL TEMPLATES---
+{
+	"BRANCH": "master",
+	"BUILD_NUMBER": "12",
+	"COMMIT": "4923x0c3380413ec9288e3c0bfbf534b0f18fed1",
+	"TAG": "",
+	"cluster": "my-gke-cluster",
+	"namespace": "",
+	"project": "my-gcp-proj",
+	"zone": "us-east1-a"
+}
+---END VARIABLES AVAILABLE FOR ALL TEMPLATES---
+```
+
+## Expanding environment variables
+
+It may be desired to reference an environment variable for use in the Kubernetes manifest.
+In order to do so in `vars`, the `expand_env_vars` must be set to `true`.
+
+For example when using `drone deploy org/repo 5 production -p IMAGE_VERSION=1.0`, to get `IMAGE_VERSION` in `vars`:
+
+```yml
+expand_env_vars: true
+vars:
+  image: my-image:${IMAGE_VERSION}
```

Comment:
why does this use `${}` and the other example uses `$${}`?

Reply:
Good catch. It should be ``$IMAGE_VERSION`` or ``$${IMAGE_VERSION}``.

``${}`` is pre-processed by drone, so it wouldn't work.

Reply:
hence my repeated requests to explain the behavior, since I don't know what the difference between the 3 variants is.

Reply:
Since this is a drone behavior, I'd recommend linking the section on drone docs that explains it (http://docs.drone.io/environment/).

Reply:
Added details, hope that clears things up.